diagrams/src/main.js

```diff
@@ -15,8 +15,8 @@
     return {
         drawDiagramsFromParserInstance: function(parserInstanceToDraw, targetDiv) {
             var topRules = parserInstanceToDraw.getGAstProductions().values()
-            targetDiv.innerHTML = diagrams_builder.buildSyntaxDiagramsText(topRules);
-            diagrams_behavior.initDiagramsBehavior()
+            targetDiv.innerHTML = builder.buildSyntaxDiagramsText(topRules);
+            builder.initDiagramsBehavior()
```

Comment:
Sorry, didn't get a chance to fully test.  Working on getting a bundle built but haven't finished.  Just wanted to publish up the obvious globals I ran into.